models/articles.models.js

```diff
@@ -21,9 +21,10 @@ exports.selectArticleById = (article_id) => {
 	});
 };
 
-exports.selectArticles = (sortby, order) => {
+exports.selectArticles = (sortby, order, filter) => {
 	const validSortBy = { created_at: 'created_at' };
 	const validOrder = { asc: 'ASC', desc: 'DESC' };
+	const validfilter = { cats: 'cats' };
```

Comment:
hardcoding the valid topics won't work for too long - what if someone adds a topic?